src/frontend/citizen-portal/nginx.conf

```diff
@@ -30,3 +38,6 @@ location /nginx_status {
 location /api/ {
     proxy_pass http://citizen-api/api/;
 }
+
+server_tokens off;
```

Comment:
Add a comment on what this does, copy a short description from the nginx docs if needed

Reply:
sure